charts/generic-prometheus-alerts/templates/rules.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.businessHoursOnly -}}
+{{- if and (.Values.businessHoursOnly) (.Values.applicationAlertsEnabled) -}}
```

Comment:
I added `applicationAlertsEnabled` previously (default is `true`) so that we have prometheus alerts in environments that are a mix of cloud platform and azure, however missed this one out.  Needs excluding as well if alerts are off since there is no `PrometheusRule` resource in the azure kubernetes cluster.